lib/api.rb

```diff
@@ -31,7 +31,7 @@ def get_works(params = {})
 
         url = "#{ENV['API_URL']}/dois?" + URI.encode_www_form(params)
       end
-      response = Maremma.get(url, headers: { "Content-Type" => "application/json;charset=UTF-8" }, timeout: TIMEOUT)
+      response = Maremma.get(url, headers: { "Content-Type" => "application/json;charset=UTF-8", "Accept-Encoding"=> "gzip" }, timeout: TIMEOUT)
```

Comment:
@mfenner FYI, AS part of this change, I re-added gzip compression in Search for all api calls.  I think that's fine, or is there any other reason to not use it??